docs/DeveloperGuide.md

```diff
@@ -154,6 +154,44 @@ Classes used by multiple components are in the `seedu.addressbook.commons` packa
 
 This section describes some noteworthy details on how certain features are implemented.
 
+### Increment Feature
+
+#### Implementation
+
+The increment feature is facilitated by the following operations:
+* `Model#getFilteredPersonList()` — Returns the list of employees in the filtered list.
```

Comment:
small update to the method; 
Model#getFilteredPersonList() has been changed to Model#getSortedFilteredPersonList()

Reply:
Do remember to change the method name downstream also